### ISSUE TITLE
Missing packages

### ISSUE THREAD

**Issue by lucasgautheron** [ActionFPS/ActionFPS-Game#169]
# Bug template

Some textures are missing in the v1.2.0.2 package.

```
couldn't load texture packages/misc/chars_rounded.png
couldn't load texture packages/misc/radaricons.png
```

Then
```
couldn't load texture packages/misc/radaricons.png
couldn't load texture packages/misc/teamicons.png
```

## Steps to reproduce behavior

Install and run the game.

**Comment by ScalaWilliam** [comment]
So these media are unique to AF or what?

**Comment by lucasgautheron** [comment]
I guess they were brought to AssaultCube master branch after v1.2.0.2 was released

**Comment by ScalaWilliam** [comment]
Can't see chars_rounded anywhere here:
- https://github.com/assaultcube/AC/tree/next/packages/misc
- https://github.com/assaultcube/AC/tree/master/packages/misc

**Comment by lucasgautheron** [comment]
this is because chars_serif is default in AF and thus remained as chars_default.
the previous default had to be renamed to chars_rounded

**Comment by ScalaWilliam** [comment]
All right. It is possible to rename this file on installation time. How would you propose to solve this one?

**Comment by ScalaWilliam** [comment]
Causes nasty error messages on Win 10!
Now sounds more important 📦 
![image](https://cloud.githubusercontent.com/assets/2464813/25776880/cc142a36-32fd-11e7-8ad4-21f63e08169c.png)

**Comment by lucasgautheron** [comment]
I suggest the following:
 * Modified packages should use new names
 *Host a repository for "our" packages, that should be installed on top of the official set.

**Comment by ScalaWilliam** [comment]
Yes I can make that happen. I'm not sure what of "our" is not really our (ie copyrighted) though, from that dataset.
